### ISSUE TITLE
Hyperparameters

### ISSUE THREAD

**Issue by YovaKem** [ufal/augpt#2]
Hi, can you specify what is meant by: "In this case the expected number of GPUs is four, you may need to adjust learning_rate and/or gradient-accumulation-steps accordingly." I would like to replicate the training conditions you used as closely as possible, so I will use 4 GPUs and wanted to check what the right lr and gradient accumulation values should be.

**Comment by jkulhanek** [comment]
Hi, with four GPUs, use the default hyperparameters.

**Comment by YovaKem** [comment]
Ah okay, I think I misread the sentence cited above. Thanks!
